### PR TITLE
test(divider): added tests

### PR DESCRIPTION
**Pull request by Lunkan89**
## **Describe pull-request**  
Added tests from divider component

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** [CDEP-3026](https://tegel.atlassian.net/browse/CDEP-3026)

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. fetch and checkout branch `fix/CDEP-3026-divider-tests`
2. in root run: `npm run test`

## **Checklist before submission**
- [ ] I have added unit tests for my changes (if applicable)
- [x] All existing tests pass
- [ ] I have updated the documentation (if applicable)

[CDEP-3026]: https://tegel.atlassian.net/browse/CDEP-3026?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by theJohnnyMe — APPROVED** [review]
Nice one, checked on my local. All tests passing ;)

**Comment by timrombergjakobsson** [comment]
Looks great! Should we add dark-mode tests while we are at it?

**Comment by nathalielindqvist** [comment]
> Looks great! Should we add dark-mode tests while we are at it?

I'm not sure we have tests for dark-mode for any of the other components? Do we want this to be the first in that case or should we wait with it?

**Comment by timrombergjakobsson** [comment]
> > Looks great! Should we add dark-mode tests while we are at it?
> 
> I'm not sure we have tests for dark-mode for any of the other components? Do we want this to be the first in that case or should we wait with it?

Yes we do https://github.com/scania-digital-design-system/tegel/pull/732

File: packages/core/src/components/divider/test/darkmode/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Spinner - Standard</title>
```
Comment (nathalielindqvist) [inline packages/core/src/components/divider/test/darkmode/index.html:5]:
```suggestion
    <title>Divider- Horizontal, dark mode</title>
```

File: packages/core/src/components/divider/test/horizontal/divider.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/divider/test/horizontal/index.html';
+
+test.describe.parallel('tds-divider', () => {
+  test('expect to render a horizontal divider', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    // expect width to be greater then height
```
Comment (nathalielindqvist) [inline packages/core/src/components/divider/test/horizontal/divider.e2e.ts:10]:
```suggestion
    // expect width to be greater than height
```

File: packages/core/src/components/divider/test/horizontal/index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Spinner - Standard</title>
```
Comment (nathalielindqvist) [inline packages/core/src/components/divider/test/horizontal/index.html:5]:
```suggestion
    <title>Divider - Horizontal</title>
```

File: packages/core/src/components/divider/test/vertical/divider.e2e.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/divider/test/vertical/index.html';
+
+test.describe.parallel('tds-divider', () => {
+  test('expect to render a vertical divider', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    // expect height to be greater then width
```
Comment (nathalielindqvist) [inline packages/core/src/components/divider/test/vertical/divider.e2e.ts:10]:
```suggestion
    // expect height to be greater than width
```

File: packages/core/src/components/divider/test/vertical/index.html [hunk 5]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Spinner - Standard</title>
```
Comment (nathalielindqvist) [inline packages/core/src/components/divider/test/vertical/index.html:5]:
```suggestion
    <title>Divider - Vertical</title>
```

**Review by nathalielindqvist — APPROVED** [review]
Great job! Left a few suggestions on title changes, otherwise LGTM!

**Pull request merged**